src/app/routines/exercisesTable/ExercisesTable.tsx

```diff
@@ -0,0 +1,38 @@
+import { TableContainer, Table, TableHead, TableRow, TableCell, TableBody } from '@mui/material';
+import ExercisesTableRow from './ExercisesTableRow';
+import { useRoutineExercises } from '@/app/common/context/RoutineExercisesContext';
+
+interface ExerciseTableProps {
+  editable: boolean;
+}
+
+const ExercisesTable = ({ editable = true }: ExerciseTableProps) => {
+  const { exercises } = useRoutineExercises();
+
+  return (
+    <>
+      <TableContainer>
+        <Table aria-label='collapsible table'>
+          <TableHead>
+            <TableRow>
+              <TableCell>No</TableCell>
+              <TableCell>Exercise Name</TableCell>
+              <TableCell>Sets</TableCell>
+              <TableCell>Reps</TableCell>
+              <TableCell>Weight&nbsp;(unit)</TableCell>
+              <TableCell> Rest Time&nbsp;(unit)</TableCell>
```

Comment:
minor: it would be better to hardcode unit and pass it as variable, easier to change in the future